README.md

```diff
@@ -103,11 +123,10 @@ For example, if a module has versions `5.2.0-rc1`, `5.1.1`, `5.1.0`, `5.1.0-rc1`
 * `5.*-rc*` would pin the module to the latest release or prerelease of version 5, `5.2.0-rc1`. In order to use
 prerelease versions, the version *must* contain a `-` prerelease prefix.
 
-## Path
-
-The `Path` property is optional. By default, the module is saved to a `PSModules` directory in the same directory as
-the "pxget.json" file. If the `Path` property is provided, the module will instead be saved to that directory, which
-will be created if it doesn't exist. Relative paths are resolved from the directory of the "pxget.json" file.
+Modules are pinned/locked to a specific version of a module the first time `pxget install` is run. The command generates
+a `pxget.lock.json` file, with the specific versions of each module to install. Once a lock file is created, `pxget
+install` will only install the module versions listed in the lock file. To update the versions in the lock file to
+newer versions or to reflect changes made to the `pxget.json` file, run `pxget lock`.

```

Comment:
I'd prefer the command that updates the lock file be `pxget update`. If I already have a `pxget.lock.json` file then it's not immediately apparent what the `pxget lock` command would do, if anything at all. To me it sounds like the `pxget lock` command would generate a `pxget.lock.json` file from the versions of module already on on-disk in the `Modules` directory, i.e. lock/resolve without calling `Find-Module` and talking to a remote registry.

Whereas `pxget update` is intuitive that it would update the lock file with the latest versions of modules in `pxget.json`

Reply:
I don't like the `update` command. To me its always ambiguous what I'm updating. The locked version? Or the version in my configuration file? I want to eventually have a command to do each: update the locked version and update the version in the pxget.json file.

Maybe `update-lock`? `relock` ? Other?

Reply:
To me it makes sense that the `update` command would operate on the lock file, updating all the modules to the latest version of their defined version pins. You're almost always going to pin a module version with a wildcard, so I don't think it makes much sense to have the `update` command update a wildcard version in `pxget.json`.

But if a command were to update the `pxget.json` file I would expect it to be the `install` command. Called in its simplest form, `pxget install`, it simply installs modules per the lock file. But give it a module name and version, `pxget install Whiskey 1.*`, and it will add that module to the `pxget.json` file, generate the lock file, and install all the modules. This gives the user an easy way to add (and remove) modules from `pxget.json` via pxget commands without having to manually edit the JSON. Or if you want to be explicit about it, use a switch/flag that indicates the module should be added to `pxget.json`, e.g. `pxget install Whiskey 1.* --save`

Otherwise, I'd simply vote for the command to be called `update-lock`.

Reply:
I changed the command to `update`.